docs/examples/managed/GoogleDemo.ipynb

```diff
@@ -39,9 +39,11 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from llama_index import LLMPredictor
</pre><p>btw llm_predictor has been deprecated </p><p>I think you can directly set lm=llm where llm is Gemini()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9824/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>